docs/integrations/howto/datadog-logs.rst

```diff
@@ -0,0 +1,65 @@
+Send logs to DataDog
+====================
+
+This article will show you how to use the Aiven Rsyslog integration to send the logs from your Aiven services to DataDog.
+
+You will need:
+
+* A DataDog account.
+
+* The region to use, US or EU
```

Comment:
While other two points are clear, "The region to use, US or EU" leaves me wonder what exactly I'll need, is it that we support only these two regions? are they what we need in DataDog or on Aiven's side?

Reply:
With DataDog, you have to actually register in one region or the other. Let me try to make that clearer

Reply:
Updated, hopefully makes more sense.